### DISCUSSION TITLE
Does it work behind VPN ?

### DISCUSSION THREAD

**Discussion by rthangam** [community/community#7818] [Copilot Conversations]
github copilot extension works fine for me on VS Code but it doesn't work when i use work VPN. How to make it work inside VPN ? Any suggestion ?

**Comment by TarunavBA** [comment]
I also would like it to work behind a vpn!

**Comment by MulleDK19** [comment]
Proxies are not supported atm.

**Reply by AlexanderLanin** [reply]
That would match #7704

**Reply by rthangam** [reply]
Any plans to support VPN in future ?
